.github/workflows/apidiff.yml

```diff
@@ -100,3 +130,39 @@ jobs:
             repo: context.repo.repo,
             labels: ['breaking change']
           })
+    # Step 4: For non-GA APIs, post the breaking change details as a comment.
+    # This step only runs if the ga_check step succeeded, meaning a breaking
+    # change was found in a non-GA API. This provides high-visibility feedback
+    # without blocking the PR, and avoids redundant comments for GA APIs where
+    # the failed presubmit check is the primary signal.
+    - name: Post breaking change details as a comment
+      if: ${{ steps.ga_check.outcome == 'success' && !github.event.pull_request.head.repo.fork }}
```

Comment:
Doesn't this need `steps.detect.outputs.breaking_change_found == 'true'` as well? Othewise I'd expect it to *always* claim that there's a breaking change, even if there isn't.

Reply:
Ah right. Sure, that makes sense. I think - not a GHA expert though!